pkgs/development/python-modules/dm-tree/default.nix

```diff
@@ -5,13 +5,25 @@
   attrs,
   buildPythonPackage,
   cmake,
+  fetchpatch,
   fetchFromGitHub,
   lib,
   numpy,
   pybind11,
   wrapt,
 }:
-
+let
+  patchCMakeAbseil = fetchpatch {
+    name = "0001-don-t-rebuild-abseil.patch";
+    url = "https://raw.githubusercontent.com/conda-forge/dm-tree-feedstock/93a91aa2c13240cecf88133e2885ade9121b464a/recipe/patches/0001-don-t-rebuild-abseil.patch";
+    sha256 = "08fqygfwpi2645079mv0s6d1q6psy50rg7q1l90r42f1l40z4acq";
```

Comment:
using `hash` instead of `sha256` is preferred. we're trying to deprecate `sha256`

Reply:
Done